plugins/eos/gs-plugin-eos.c

```diff
@@ -744,6 +766,19 @@ app_is_banned_for_personality (GsPlugin *plugin, GsApp *app)
 }
 
 static gboolean
+app_is_banned_coding_app (GsPlugin *plugin, GsApp *app)
+{
+	GsPluginData *priv = gs_plugin_get_data (plugin);
+	const char *id = gs_app_get_id (app);
+
+	/* the coding chatbox is in the core ostree, so unlike the
+	   personality-based blocking, we block even if it is installed */
+
+	return ((g_strcmp0 (priv->product, "coding") != 0) &&
+	        (g_strcmp0 (id, "com.endlessm.Coding.Chatbox.desktop") == 0));
+}
```

Comment:
Interestingly enough, in the shell we blacklist based on the `enable-coding-game` key in `org.gnome.Shell`. Would it make more sense to blacklist based on that?

Reply:
Yes, agreed, and I have updated the PR.